Example/iOS Example/WillowConfiguration.swift

```diff
@@ -35,14 +35,16 @@ struct WillowConfiguration {
     // MARK: - Formatters
 
     private struct PrefixFormatter: Formatter {
-        let prefix: String
+        let emoticons: String
+        let name: String
 
-        init(prefix: String) {
-            self.prefix = prefix
+        init(emoticons: String, name: String) {
```

Comment:
emoji would be a more accurate term here rather than emoticon. I looked it up to verify 😀